lib/ex_dtls.ex

```diff
@@ -38,24 +38,30 @@ defmodule ExDTLS do
   Initializes `ExDTLS`.
 
   Accepts a keyword list with the following options (`t:opts_t/0`):
-  * `client_mode` - `true` if ExDTLS module should work as a client or `false` if as a server
-  * `dtls_srtp` - `true` if DTLS-SRTP handshake should be performed or `false` if a normal one
-  * `pkey` - private key to use in this SSL context. Must correspond to `cert`
-  * `cert` - certificate to use in this SSL context. Must correspond to `pkey`
-
+  * `client_mode` - `true` if ExDTLS module should work as a client or `false` if as a server.
+  This option is required.
```

Comment:
Shouldn't this be a positional argument? I mean, it isn't really an option if it's required. I would imagine something like `ExDTLS.start_link(:client | :server)`.

Reply:
Sometimes it's more readable to write `ExDTLS.init(mode: server)` than `ExDTLS.init(:server)`. That's also what Broadway does (they require name) and what `NimbleOptions` allows for. I will change the name from `client_mode :: boolean()` to `mode: :server | :client`.